light-system-programs/tests/verifier_tests.ts

```diff
@@ -342,6 +339,7 @@ describe("Verifier Zero and One Tests", () => {
         tmp_tx.transactionInputs.publicInputs.inputNullifier[i] = new Array(
           32,
         ).fill(2);
+        await sleep(SLEEP_BUFFER);
```

Comment:
as discussed we could make the sleep depend on the platform.
This is how detect the platform in the cli:
https://github.com/Lightprotocol/light-protocol/blob/main/cli/src/psp-utils/download.ts#L36

Reply:
👍 for now just importing from psp-utils so we don't have dupe code but probably makes sense to put this separately at some point? any ideas?

Reply:
update: moved to util in zkjs